packages/transit-vehicle-overlay/src/VehicleMarker/styled.js

```diff
@@ -85,49 +90,49 @@ export const TrackedRail = styled(NormRail)`
  * TODO: both icon names and these modes need to align better to standards
  * TODO: icons using trimet stuff needs to get away from MAX / WES / AERIALTRAM names, etc...
  */
-export function makeVehicleIcon(mode, selectColor, isTracked) {
+export function makeVehicleIcon(mode, color, highlightColor, isTracked) {
```

Comment:
This function `makeVehicleIcon` should be in a separate file IMO because it doesn't have to do with styling other components.

Reply:
Hey Binh ... yeah, good catch.  I moved that function over to utils.js.